modules/storage-create-storage-class-console.adoc

```diff
@@ -0,0 +1,30 @@
+:_content-type: PROCEDURE
+[id="storage-create-storage-class-console_{context}"]
+= Creating the {StorageClass} storage class using web console
```

Comment:
Suggestions: 

1. add "the" before console - "using the web console"
2. should "object" be added after storage class? (as in other comment)